i18n_alchemy.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.2.0'
 
-  s.add_dependency "activesupport", ">= 4.0.0", "< 7.0"
+  s.add_dependency "activesupport", ">= 4.0.0", "<= 7.0"
   s.add_dependency "i18n", ">= 0.7"
 
   s.add_development_dependency "actionpack", ">= 4.0.0", "< 7.0"
```

Comment:
It looks like we need to open up the dependency checks here to <= 7.0 (both actionpack & activerecord), otherwise we get error constraints.

Reply:
I forgot this! :sweat_smile: